### PR TITLE
JES in-command write_lines fixup. Closes #1906

### LINKED ISSUE #1906: write_lines(Array[File]) creates file with unlocalized file paths
When using write_lines() on an Array[File] object, the paths in the resulting file are the original file paths rather than the localized ones, causing tasks to fail.

I have the following WDL:
```
task aggregate_mafs {
    Array[File]+ mafpaths
    String out_prefix
    String? out_suffix

    command {
        python /src/Merge_MAFs.py ${"--suffix " + out_suffix} ${out_prefix} ${write_lines(mafpaths)}
    }

    output {
        File aggregated_maf = "${out_prefix}${'.' + out_suffix}.maf"
    }

    runtime {
        docker : "broadgdac/aggregate_mafs:2"
    }

    parameter_meta {
        mafpaths : "File containing paths to MAF files."
        out_prefix : "Identifying prefix. Usually the set entity this is run on."
        out_suffix : "Descriptive text to precede the .maf suffix."
    }
}

workflow aggregate_mafs_workflow {
    call aggregate_mafs
}
```

With the following inputs.json:
```js
{
  "aggregate_mafs_workflow.aggregate_mafs.out_suffix": "test",
  "aggregate_mafs_workflow.aggregate_mafs.out_prefix": "ACC",
  "aggregate_mafs_workflow.aggregate_mafs.mafpaths": ["tests/TCGA-OR-A5J1-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5J2-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5J3-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5J4-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5J5-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5J6-01A-31D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5J7-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5J8-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5J9-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JA-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JB-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JC-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JD-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JE-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JF-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JG-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JH-01A-11D-A30A-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JI-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JJ-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JK-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JL-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JM-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JO-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JP-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JQ-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JR-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JS-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JT-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JU-01A-11D-A30A-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JV-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JW-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JX-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JY-01A-31D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5JZ-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5K0-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5K1-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5K2-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5K3-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5K4-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5K5-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5K6-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5K8-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5K9-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5KB-01A-11D-A30A-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5KO-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5KP-01A-11D-A30A-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5KQ-01A-11D-A30A-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5KS-01A-11D-A30A-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5KT-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5KU-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5KV-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5KW-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5KX-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5KY-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5KZ-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5L1-01A-11D-A30A-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5L2-01A-11D-A30A-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5L3-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5L4-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5L5-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5L6-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5L8-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5L9-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LA-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LB-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LC-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LD-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LE-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LF-01A-11D-A30A-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LG-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LH-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LI-01A-11D-A30A-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LJ-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LK-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LL-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LM-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LN-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LO-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LP-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LR-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LS-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OR-A5LT-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-OU-A5PI-01A-12D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-P6-A5OF-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-P6-A5OG-01A-22D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-P6-A5OH-01A-11D-A30A-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-PA-A5YG-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-PK-A5H8-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-PK-A5H9-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-PK-A5HA-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-PK-A5HB-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt","tests/TCGA-PK-A5HC-01A-11D-A30A-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt"]
}
```

Running the script:
```
$ java -jar ~/bin/cromwell-24.jar run /Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/aggregate_mafs.wdl tests/inputs.json
[2017-01-20 09:31:10,44] [info] Slf4jLogger started
[2017-01-20 09:31:10,52] [info] RUN sub-command
[2017-01-20 09:31:10,52] [info]   WDL file: /Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/aggregate_mafs.wdl
[2017-01-20 09:31:10,52] [info]   Inputs: /Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/tests/inputs.json
[2017-01-20 09:31:10,58] [info] SingleWorkflowRunnerActor: Submitting workflow
[2017-01-20 09:31:10,63] [info] Workflow 814c47aa-9d11-4c81-a08c-f2b77c002b46 submitted.
[2017-01-20 09:31:10,63] [info] SingleWorkflowRunnerActor: Workflow submitted 814c47aa-9d11-4c81-a08c-f2b77c002b46
[2017-01-20 09:31:11,29] [info] Running with database db.url = jdbc:hsqldb:mem:396f6af4-b493-451b-ad19-2042625bf63e;shutdown=false;hsqldb.tx=mvcc
[2017-01-20 09:31:15,99] [info] Running migration RenameWorkflowOptionsInMetadata with a read batch size of 100000 and a write batch size of 100000
[2017-01-20 09:31:16,00] [info] [RenameWorkflowOptionsInMetadata] 100%
[2017-01-20 09:31:16,03] [info] Metadata summary refreshing every 2 seconds.
[2017-01-20 09:31:16,06] [info] 1 new workflows fetched
[2017-01-20 09:31:16,06] [info] WorkflowManagerActor Starting workflow 814c47aa-9d11-4c81-a08c-f2b77c002b46
[2017-01-20 09:31:16,06] [info] WorkflowManagerActor Successfully started WorkflowActor-814c47aa-9d11-4c81-a08c-f2b77c002b46
[2017-01-20 09:31:16,06] [info] Retrieved 1 workflows from the WorkflowStoreActor
[2017-01-20 09:31:16,30] [info] MaterializeWorkflowDescriptorActor [814c47aa]: Call-to-Backend assignments: aggregate_mafs_workflow.aggregate_mafs -> Local
[2017-01-20 09:31:16,48] [info] WorkflowExecutionActor-814c47aa-9d11-4c81-a08c-f2b77c002b46 [814c47aa]: Starting calls: aggregate_mafs_workflow.aggregate_mafs:NA:1
[2017-01-20 09:31:16,69] [info] BackgroundConfigAsyncJobExecutionActor [814c47aaaggregate_mafs_workflow.aggregate_mafs:NA:1]: python /src/Merge_MAFs.py --suffix test ACC /root/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/execution/wdlarray-8012c813f9ee08681a0b1fb427a70b0d.tmp
[2017-01-20 09:31:16,70] [info] BackgroundConfigAsyncJobExecutionActor [814c47aaaggregate_mafs_workflow.aggregate_mafs:NA:1]: executing: docker run --rm -v /Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs:/root/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs -i broadgdac/aggregate_mafs:2 /bin/bash /root/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/execution/script
[2017-01-20 09:31:16,71] [info] BackgroundConfigAsyncJobExecutionActor [814c47aaaggregate_mafs_workflow.aggregate_mafs:NA:1]: command: "/bin/bash" "/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/execution/script.submit"
[2017-01-20 09:31:16,78] [info] BackgroundConfigAsyncJobExecutionActor [814c47aaaggregate_mafs_workflow.aggregate_mafs:NA:1]: job id: 2329
[2017-01-20 09:31:16,79] [info] BackgroundConfigAsyncJobExecutionActor [814c47aaaggregate_mafs_workflow.aggregate_mafs:NA:1]: BackgroundConfigAsyncJobExecutionActor [814c47aa:aggregate_mafs_workflow.aggregate_mafs:NA:1] Status change from - to SharedFileSystemRunStatus(false)
[2017-01-20 09:33:07,55] [info] BackgroundConfigAsyncJobExecutionActor [814c47aaaggregate_mafs_workflow.aggregate_mafs:NA:1]: BackgroundConfigAsyncJobExecutionActor [814c47aa:aggregate_mafs_workflow.aggregate_mafs:NA:1] Status change from SharedFileSystemRunStatus(false) to SharedFileSystemRunStatus(true)
[2017-01-20 09:33:07,58] [info] Message [cromwell.subworkflowstore.SubWorkflowStoreActor$SubWorkflowStoreCompleteSuccess] from Actor[akka://cromwell-system/user/SingleWorkflowRunnerActor/$b#-910401033] to Actor[akka://cromwell-system/user/SingleWorkflowRunnerActor/WorkflowManagerActor/WorkflowActor-814c47aa-9d11-4c81-a08c-f2b77c002b46#617869376] was not delivered. [1] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[2017-01-20 09:33:07,58] [error] WorkflowManagerActor Workflow 814c47aa-9d11-4c81-a08c-f2b77c002b46 failed (during ExecutingWorkflowState): Call aggregate_mafs_workflow.aggregate_mafs:NA:1: return code was 1
java.lang.RuntimeException: Call aggregate_mafs_workflow.aggregate_mafs:NA:1: return code was 1
	at cromwell.backend.standard.StandardAsyncExecutionActor$class.handleExecutionResult(StandardAsyncExecutionActor.scala:432)
	at cromwell.backend.impl.sfs.config.BackgroundConfigAsyncJobExecutionActor.handleExecutionResult(ConfigAsyncJobExecutionActor.scala:113)
	at cromwell.backend.standard.StandardAsyncExecutionActor$class.handlePollSuccess(StandardAsyncExecutionActor.scala:370)
	at cromwell.backend.impl.sfs.config.BackgroundConfigAsyncJobExecutionActor.handlePollSuccess(ConfigAsyncJobExecutionActor.scala:113)
	at cromwell.backend.standard.StandardAsyncExecutionActor$$anonfun$poll$2.apply(StandardAsyncExecutionActor.scala:333)
	at cromwell.backend.standard.StandardAsyncExecutionActor$$anonfun$poll$2.apply(StandardAsyncExecutionActor.scala:332)
	at scala.util.Success$$anonfun$map$1.apply(Try.scala:237)
	at scala.util.Try$.apply(Try.scala:192)
	at scala.util.Success.map(Try.scala:237)
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:237)
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:237)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply$mcV$sp(BatchingExecutor.scala:91)
	at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply(BatchingExecutor.scala:91)
	at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply(BatchingExecutor.scala:91)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:72)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:90)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:39)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:415)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)

[2017-01-20 09:33:07,58] [info] WorkflowManagerActor WorkflowActor-814c47aa-9d11-4c81-a08c-f2b77c002b46 is in a terminal state: WorkflowFailedState
[2017-01-20 09:33:24,62] [info] SingleWorkflowRunnerActor workflow finished with status 'Failed'.
Workflow 814c47aa-9d11-4c81-a08c-f2b77c002b46 transitioned to state Failed
```

Failure is due to inability to find the files listed in the file created by write_lines():
```
$ tail -n 10 stderr
Traceback (most recent call last):
  File "/src/Merge_MAFs.py", line 182, in <module>
    main(sys.argv[1:])
  File "/src/Merge_MAFs.py", line 76, in main
    concatenatedMafFilename = _handle_mafs(args)
  File "/src/Merge_MAFs.py", line 83, in _handle_mafs
    mafPaths = _getMafPaths(args.mafpaths)
  File "/src/Merge_MAFs.py", line 98, in _getMafPaths
    raise Exception("MAF doesn't exist: %s" % mafPath)
Exception: MAF doesn't exist: /Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/inputs/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/tests/TCGA-OR-A5J1-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt
```

File created by write_lines() - I believe the files listed within *should* be under the directory ```/root/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/inputs/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/tests```, instead they are all listed at their system path, ```/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/inputs/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/tests```:
```
$ head cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/execution/wdlarray-8012c813f9ee08681a0b1fb427a70b0d.tmp 
/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/inputs/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/tests/TCGA-OR-A5J1-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt
/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/inputs/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/tests/TCGA-OR-A5J2-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt
/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/inputs/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/tests/TCGA-OR-A5J3-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt
/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/inputs/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/tests/TCGA-OR-A5J4-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt
/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/inputs/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/tests/TCGA-OR-A5J5-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt
/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/inputs/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/tests/TCGA-OR-A5J6-01A-31D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt
/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/inputs/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/tests/TCGA-OR-A5J7-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt
/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/inputs/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/tests/TCGA-OR-A5J8-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt
/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/inputs/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/tests/TCGA-OR-A5J9-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt
/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/cromwell-executions/aggregate_mafs_workflow/814c47aa-9d11-4c81-a08c-f2b77c002b46/call-aggregate_mafs/inputs/Users/dheiman/Documents/workspace/gdac-firecloud/tasks/aggregate_mafs/tests/TCGA-OR-A5JA-01A-11D-A29I-10.ff872fc4-bd1c-4975-85c8-3655ccd199a2.maf.txt
```

Apologies if this was corrected by #1857, I originally included this in #1875 last week, but had not heard back about this part of that issue.

---- 

Update: `write_lines()` was improved for Cromwell 27 but some issues appear to remain.

### PR DESCRIPTION
**Pull request by cjllanwarne**
In lieu of a real fix to `write_lines`'s odd behaviour, this PR at least brings it into consistency with the other backends

### CODE REVIEW

File: supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala [hunk 1]
```diff
@@ -145,20 +145,25 @@ class JesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
     }
   }
 
-  private[jes] def generateJesInputs(jobDescriptor: BackendJobDescriptor): Set[JesInput] = {
+  private[jes] def generateJesInputs(jobDescriptor: BackendJobDescriptor): Try[Set[JesInput]] = {
 
-    val writeFunctionFiles = call.task.evaluateFilesFromCommand(jobDescriptor.fullyQualifiedInputs, backendEngineFunctions) map {
-      case (expression, file) =>  expression.toWdlString.md5SumShort -> Seq(file)
-    }
+    commandLinePreProcessor(jobDescriptor.inputDeclarations) map { preprocessedInputs =>
```
Comment (cjllanwarne) [inline supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala]: I think you're correct. I went belt-and-braces by applying both preProcessor and the valueMapper but for JES only the valueMapper actually does anything.

Good spot!

**Comment by Horneth** [comment]
👍  after Miguel's comment is addressed !

**Comment by mcovarr** [comment]
:+1: after de-nooping and centauring

File: .travis.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ before_install:
   - openssl aes-256-cbc -K "$encrypted_5ebd3ff04788_key" -iv "$encrypted_5ebd3ff04788_iv" -in src/bin/travis/resources/jesConf.tar.enc -out jesConf.tar -d || true
 env:
   global:
-    - CENTAUR_BRANCH=develop
+    - CENTAUR_BRANCH=cjl_write_lines
```
Comment (cjllanwarne) [inline .travis.yml]: Remember to revert!

**Pull request merged**